scram/route_manager/tests/test_swagger.py

```diff
@@ -0,0 +1,28 @@
+"""This file contains tests for the swagger API endpoints."""
+
+import pytest
+from django.urls import reverse
+
+
+@pytest.mark.django_db
+def test_swagger_api(client):
+    """Test that the Swagger API endpoint returns a successful response."""
+    url = reverse("swagger-ui")
+    response = client.get(url)
```

Comment:
Can you check that the response contains something to ensure that swagger loaded the API? e.g. `"Entry"`

Reply:
My first attempts are big fails. It looks like the client response only holds the HTML and the redoc and swagger stuff is all done via JS. That starts to get into selenium type territory which I'd rather not do.